.github/workflows/merge-flow.yaml

```diff
@@ -94,6 +94,11 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Remove dependabot configuration
+        continue-on-error: true
+        run: |
+          rm -fr .github/dependabot.*
```

Comment:
Even though there is no difference in command I think `rm -rf` is better to read :)

Reply:
do you need a `-r` ?

Reply:
`-r` is not required. I've fixed it. Thanks @sthaha and @slashpai !